common/persistence/nosql/factory.go

```diff
@@ -149,7 +150,7 @@ func (f *executionStoreFactory) close() {
 
 // new implements ExecutionStoreFactory interface
 func (f *executionStoreFactory) new(shardID int) (p.ExecutionStore, error) {
-	pmgr, err := NewWorkflowExecutionPersistence(shardID, f.client, f.session, f.logger)
+	pmgr, err := cassandra2.NewWorkflowExecutionPersistence(shardID, f.client, f.session, f.logger)
```

Comment:
can this be injected? Or this will be refactored in the future?

Reply:
Good catch. It will be refactored and remove very soon. 
I rename it to cassandraOld and added a comment.